lib/stripe/stripe_object.rb

```diff
@@ -370,6 +369,10 @@ class << self; self; end
         return mth.call(args[0])
       elsif @values.key?(name)
         return @values[name]
+      elsif name.to_s.end_with?("?") && @values.key?(name.to_s[0...-1]&.to_sym)
```

Comment:
Could we mark our changes clearly with silly comments of the form
```
# [START CM CHANGE]
# was:
# old stuff
...new stuff
# [END CM CHANGE]
```
or smth to that effect?
It may help if we merge newer versions with a diff tool